livekit-agents/livekit/agents/llm/function_context.py

```diff
@@ -303,3 +304,96 @@ def _is_optional_type(typ) -> Tuple[bool, Any]:
             return True, non_none_args[0]
 
     return False, None
+
+
+def create_ai_function_info(
+    fnc_ctx: FunctionContext,
+    tool_call_id: str,
+    fnc_name: str,
+    raw_arguments: str,  # JSON string
+) -> FunctionCallInfo:
+    if fnc_name not in fnc_ctx.ai_functions:
```

Comment:
This function was initially for OpenAI but if they all support the same kind of arguments (same json format).
Then it makes sense to move it.

Tho I'm not sure about what the implementation will looks like when we add support for structured output

Reply:
I would keep this function prefixed with _ tho. It's not really relevant to users